nav2_map_server/CMakeLists.txt

```diff
@@ -68,6 +69,25 @@ install(TARGETS ${executable_name} ${library_name}
   LIBRARY DESTINATION lib
   RUNTIME DESTINATION lib/${PROJECT_NAME}
 )
+set(executable_name map_saver)
```

Comment:
Can we use different variable names here instead? eg Instead of `executable_name` use `map_server_executable_name` and `map_saver_executable_name`. Same for `dependencies`. Or we could just get rid of the variable and spell things out.

Then we could keep the structure of the cmakelists like the others -eg define the target and dependencies at the top, install them at the bottom.

Reply:
Changing to map_saver_executable and map_saver_dependencies

Reply:
refactored the CMakeLists to try and meet your request. See latest commits